src/Algebra/Properties/KleeneAlgebra.agda

```diff
@@ -0,0 +1,24 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some derivable properties
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra.Bundles
+
+module Algebra.Properties.KleeneAlgebra {k₁ k₂} (K : KleeneAlgebra k₁ k₂) where
+
+open KleeneAlgebra K
+open import Algebra.Definitions _≈_
+open import Relation.Binary.Reasoning.Setoid setoid
+open import Function
```

Comment:
Why do you need `Function`?

Reply:
Ah sorry I was trying something. I'll remove it.